packages/api/amplify_api/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.0-next.1+1
+
+### Fixes
+- fix(api): allow setting extra GQL request params from model helpers ([#2423](https://github.com/aws-amplify/amplify-flutter/pull/2423))
+- fix(api): fix subscription race condition from previous connection close ([#2405](https://github.com/aws-amplify/amplify-flutter/pull/2405))
+
```

Comment:
request: add this one too https://github.com/aws-amplify/amplify-flutter/pull/2413